### PR TITLE
Autonomous

### PR DESCRIPTION
**Pull request by mastermind-mayhem**
Mobility
Far Mobility
Charge
Mobility and Charge
Drop down

### CODE REVIEW

**Review by awesomeatpiano — APPROVED** [review]

**Review by Banana123abcd — APPROVED** [review]
looks good

**Review by End10 — APPROVED** [review]
big-smile-emoticon-with-thumbs-up-vector-23657760

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -108,27 +118,27 @@ public RobotContainer() {
...
+    // supplier(6, joysticks.DRIVER).onTrue(rotRight90Cmd);
+
+    // // Arm Extension In
+    // supplier(5, joysticks.DRIVER).onTrue(armIn).onFalse(armExtStop);
+    // // Arm Extension Out
+    // supplier(3, joysticks.DRIVER).onTrue(armOut).onFalse(armExtStop);
+    // // Arm Rotation Up
+    // supplier(6, joysticks.DRIVER).onTrue(armUp).onFalse(armRotStop);
+    // // Arm Rotation Down
+    // supplier(4, joysticks.DRIVER).onTrue(armDown).onFalse(armRotStop);
```
Comment (mathyourlife) [inline src/main/java/frc/robot/RobotContainer.java:141]: don't comment this out

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -55,12 +65,6 @@ public class RobotContainer {
   private final RotRightCmd rotRightCmd = new RotRightCmd(wristSubsystem);
   private final StopRaiseCmd stopRaiseCmd = new StopRaiseCmd(wristSubsystem);
   private final StopRotCmd stopRotCmd = new StopRotCmd(wristSubsystem);
-  private final ClawSubsystem clawSubsystem = new ClawSubsystem();
-  private final ClampCmd ClampCmd = new ClampCmd(clawSubsystem);
-  private final ReleaseCmd ReleaseCmd = new ReleaseCmd(clawSubsystem);
-  private final ShiftLeftCmd ShiftLeftCmd = new ShiftLeftCmd(clawSubsystem);
-  private final ShiftRightCmd ShiftRightCmd = new ShiftRightCmd(clawSubsystem);
-  private final StopClawCmd StopClawCmd = new StopClawCmd(clawSubsystem);
```
Comment (mathyourlife) [inline src/main/java/frc/robot/RobotContainer.java:63]: Why are these being deleted?

**Review by kv-gh — DISMISSED** [review]
Couple quick questions

File: src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java [hunk 3]
```diff
@@ -171,8 +171,8 @@ public void arcadeDrive(double speed, double turn) {
 
   @Override
   public void periodic() {
-    SmartDashboard.putNumber("encoder", m_encoder_Left.getPosition());
-    SmartDashboard.putNumber("angle", angles);
+    // SmartDashboard.putNumber("encoder", m_encoder_Left.getPosition());
+    // SmartDashboard.putNumber("angle", angles);
```
Comment (mathyourlife) [inline src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java:175]: Delete. don't comment out

**Review by mathyourlife — APPROVED** [review]

**Pull request merged**